core/templates/core/menu.html

```diff
@@ -1,18 +1,20 @@
 {% extends "core/base.html" %}
 {% block content %}
-<div class="row">
-    <div class="col-2 position-fixed">
-    <ul class="list-group list-group-horizontal">
-      <li class="list-group-item">An item</li>
-      <li class="list-group-item">A second item</li>
-      <li class="list-group-item">A third item</li>
-    </ul>
+
+<div class="row g-5" style="position: relative">
```

Comment:
did you mean `<div class="row lg-5" style="position: relative">`?

Reply:
Yes, but the layout seemed to be ok even with `g-5` :) so you can remove `lg-5`.